src/main/java/de/unistuttgart/chickenshockbackend/Constants.java

```diff
@@ -0,0 +1,26 @@
+package de.unistuttgart.chickenshockbackend;
+
+/**
+ * This class contains constants for defensive field checks
+ * e.g. the game time cannot be smaller than 0 seconds
+ */
+public final class Constants {
+
+    //----Game configuration attributes----
+
+    //game time in seconds
+    public static final int MIN_TIME = 0;
+    public static final int MAX_TIME = 600;
```

Comment:
Why a max time?
And shouldn't min time be 1?

Reply:
if you have 100 questions you have 6s for each... i think the values are fine

Reply:
6 seconds per question doesn't sound like a lot…

Reply:
thats the purpose of the game, and how often do you think they will put 100 questions in ONE config?

Reply:
But still, 6 seconds is **really** strict.
In that time, you'll barely read the question and all possible answers.
You haven't even navigated to the correct chicken then…
I'm still **not a friend** of limiting this at all.
I might understand an absurdly high limit, something like `60*60*6`, but everything below is simply too restrictive for no reason.

Reply:
removed it completly, only make warning in frontend.